<launch>
  <arg name="black_border" default="2"/>

  <node pkg="apriltag_ros" type="apriltag_detector_node"
	name="apriltag_detector_left" output="screen">
    <param name="refine" type="bool" value="true"/>
    <param name="family" type="int" value="0"/>
    <param name="type"   type="int" value="0"/>
    <param name="black_border" type="int" value="$(arg black_border)"/>
    <remap from="~image" to="/left/image_raw"/>
  </node>

  <node pkg="apriltag_ros" type="apriltag_detector_node"
	name="apriltag_detector_right" output="screen">
    <param name="refine" type="bool" value="true"/>
    <param name="family" type="int" value="0"/>
    <param name="type"   type="int" value="0"/>
    <param name="black_border" type="int" value="$(arg black_border)"/>
    <remap from="~image" to="/right/image_raw"/>
  </node>

</launch>
